### ISSUE TITLE
Make a pass over uncaught exceptions

### ISSUE THREAD

**Issue by Sunjammer** [fusetools/fuselibs-public#303]
https://github.com/fusetools/fuselibs-public/blob/ac501d779d803b0c0b8694690ee233233bfb23f1/Source/Fuse.Drawing/Brushes/LinearGradient.uno#L139

Playing around with stop offsets in Studio encounters this very easily, killing the preview. We should make sure as few things as possible fail in this way; At worst, this should simply make the LinearGradient not render IMO

**Comment by kusma** [comment]
Yeah, this particular case should probably use Fuse.Diagnostics.UserError() instead.

**Comment by kusma** [comment]
Changed the title to reflect what is described in the ticket.
